hw16/index.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+/*
+var admin;
+let userName;
+
+userName='Джон';
+admin=userName;
+
+console.log(admin);
+
+alert(admin)
+
+ourPlanet="Наша Планета"
+
+currentUserOfWebsite
+
+*/
+
+//TASK 1
+
+/*
+let x = 20;
+let y = 58;
+let z = 42;
+let plusResult = x + y + z
```

Comment:
забыл точку с запятой
```suggestion
let result = x + y + z;
```